Module.php

```diff
@@ -332,7 +332,8 @@ protected function corePanels()
         ];
 
         $components = Yii::$app->getComponents();
-        if (isset($components['user']['identityClass'])) {
+        // is_array() check is needed because $components['user'] could be a Closure
+        if (isset($components['user']) && is_array($components['user']) && isset($components['user']['identityClass'])) {
```

Comment:
it would, but it would not crash, so it is an improvement :)

I am open for other solutions. If instantiating the user component does not have side effects it would be fine to instantiate it and check `identityClass` property instead.